### ISSUE TITLE
Line breaks at docstring's parameters and returns section

### ISSUE THREAD

**Issue by simon-ritchie** [krassowski/jupyterlab-lsp#14]
Enhancement proposal:

If there is a line break for each argument (or return value) in the `Parameters` (or `Returns`) section of docstring, it will be easier to see.

# Example

Current docstring view :

```py
def sample_func(x, y):
    """
    Sample function.

    Parameters
    ----------
    x : int
        Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed
        do eiusmod tempor incididunt ut.
    y : int
        Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed
        do eiusmod tempor incididunt ut.

    Returns
    -------
    x : int
        Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed
        do eiusmod tempor incididunt ut.
    y : int
        Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed
        do eiusmod tempor incididunt ut.
    """
    pass
```

```py
sample_func()
```

![20190827_1](https://user-images.githubusercontent.com/25178045/63767856-b16b5200-c909-11e9-9cc5-99d74ae5b5c4.png)

I hope there is a newline before second argument (y).

e.g.

```
Parameters¶

x : int Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut.

y : int Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut.
```

**Comment by krassowski** [comment]
A temporary fix will be provided in 0.5.0 version (actually I will release it sooner, some other work went easier than I thought). There are two factors at play:
- because it seems difficult to obtain the docstrings transformed to markdown from the Python and R LSP servers (possibly this is not implemented by those servers yet)...
- I use a custom transformation to markdown as not to render otherwise unpleasant plain text; the old version was doing some basic parsing, but now I opted to put everything except for obvious code fragments into an unformated, mono-spaced code field. It looks like this:
![Screenshot from 2019-09-05 16-05-45](https://user-images.githubusercontent.com/5832902/64354676-c9bd2a00-cff7-11e9-9035-ef1da3323199.png)

Sadly the nice headers are gone for now, but I think that ideally the parsing should be obtained from LSP servers so this is a workaround anyway. If this won't be possible, I think that we could use a parsing method requested by the user, but probably as an extension to this extension, to keep this one as compatible with different kernels as possible.

**Comment by simon-ritchie** [comment]
Amazing!
I'm looking forward that version!
And I understood the current state of lsp lib.

Thanks very much.
